package.json

```diff
@@ -162,5 +162,18 @@
   },
   "devDependencies": {
     "babel-plugin-prismjs": "^2.1.0"
-  }
+  },
```

Comment:
Why added this?

Reply:
I have not added this the "," is added by vs code extension prettier